.github/workflows/validate-linux-binaries.yml

```diff
@@ -1,42 +1,49 @@
 name: Validate linux binaries
 
 on:
-  push:
-    branches:
-      main
-    paths:
-      - .github/workflows/validate-linux-binaries.yml
-      - .test/smoke_test/*
-  pull_request:
-    paths:
-      - .github/workflows/validate-linux-binaries.yml
-      - .test/smoke_test/*
```

Comment:
Whenever these files get changed, it would be important to trigger a run still?

Reply:
I moved `path` trigger into `validate-nightly-binaries.yml` and `validate-release-binaries.yml`. The reason for that is that after my change OS-specific workflows require a mandatory `channel` input, and GH, as far as I can tell, doesn't support default value for the inputs when workflow is triggered on push of pull request.

Anyway, currently, workflows for both channels and all OSs will trigger if `smoke_test` changes.